upload-chunked-file.js

```diff
@@ -0,0 +1,43 @@
+function chunkFileUpload(file, serverUrl, chunkSize) {
+    const totalChunks = Math.ceil(file.size / chunkSize);
+    let chunkIndex = 0;
+
+    (function uploadNextChunk() {
```

Comment:
Consider adding progress tracking and user feedback. This will enhance the user experience by giving information about the upload progress, which is particularly useful for large files.